.github/workflows/update-plugin-list.yml

```
@@ -27,12 +27,20 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: "3.8"
+          cache: pip
+      - name: requests-cache
```

For consistent with the other step names.

```suggestion
      - name: Requests Cache
```